lib/backend/model/keys.go

```diff
@@ -173,6 +173,14 @@ func ListOptionsIsFullyQualified(listOptions ListInterface) bool {
 	return listOptions.KeyFromDefaultPath(listOptions.defaultPathRoot()) != nil
 }
 
+// ListOptionsLastSegmentIsPrefix returns true if the final segment of the default path
+// root is a name prefix rather than the full name.
+func ListOptionsLastSegmentIsPrefix(listOptions ListInterface) bool {
```

Comment:
don't care too much about it but, I personally prefer func/methods that check something and return a bool names something like `IsLastSegmentPrefixListOptions`.